src/Calculator/Common/KeyboardShortcuManager.cs

```diff
@@ -249,6 +249,7 @@ internal static void Initialize()
                 var coreWindow = Window.Current.CoreWindow;
                 coreWindow.CharacterReceived += OnCharacterReceivedHandler;
                 coreWindow.KeyDown += OnKeyDownHandler;
+                coreWindow.KeyUp += OnKeyUpHandler;
```

Comment:
It seems unnecessary to listen to the `KeyUp` event. Since the root cause is the `CharacterReceived` event still happening after `AcceleratorKeyActivated` event, here is an easier fix by checking whether the event args have already been handled like in `OnKeyDownHandler()`.

Reply:
The value of "handled" is the same whether the user is entering normal input or if they are using an AccessKey. The calculator was attempting to perform this by disabling keyboard shortcuts while the nav pane is still open. But re-enabling keyboard shortcuts when the nav pane closes and/or during a mode change is too soon which led to the access key input being captured in the input box. So, by tracking the KeyDown/KeyUp counter we can wait until the current keyboard interaction is complete and re-enable keyboard shortcuts then.

Reply:
Friendly ping. Is this still being reviewed?

Thanks
Ryan

Reply:
Yes, It is still being reviewed

Reply:
Sorry for not being clear. I was asking about @hanzhang54, who had started looking at this. Hadn't seen a reply to jbaylon's explanation yet, so wanted to ensure this is still being looked at.

Reply:
Hi again, @hanzhang54 , do you anticipate being able to get back to reviewing this?

Thanks

Reply:
Sorry for the delay. I made a mistake on the handled event, and you are right about the same handled value whether it is normal input or the access key input. Tracking the KeyDown/KeyUp counter looks good to me.